pkg/stringutil/utils.go

```diff
@@ -74,3 +74,43 @@ func MakePathPrefixer(prefix string) func(string) string {
 		return prefix + path
 	}
 }
+
+// Test if a "s" string match a "p" pattern with joker (*, ?)
+func MatchWithJokers(p string, s string) bool {
+	runeInput := []rune(s)
+	runePattern := []rune(p)
+	lenInput := len(runeInput)
+	lenPattern := len(runePattern)
+	isMatchingMatrix := make([][]bool, lenInput+1)
+	for i := range isMatchingMatrix {
+		isMatchingMatrix[i] = make([]bool, lenPattern+1)
+	}
+	isMatchingMatrix[0][0] = true
+	for i := 1; i < lenInput; i++ {
```

Comment:
I think this loop can be eliminated as Go will initialize a slice of bool to false